### DISCUSSION TITLE
文件重命名不同步问题

### DISCUSSION THREAD

**Discussion by huarenjian1** [remotely-save/remotely-save#836] [Q&A]
在obsidian中进行的删除操作确实可以同步到另一端, 但当重命名一个文件后同步到云端, 再从云端同步到本地时, 会同时存在重命名前和重命名后的两个文件.

**Comment by huarenjian1** [comment]
PC端和手机端使用的都是双向同步, 其他设置都是默认, 用的是阿里云的S3

**Comment by fyears** [comment]
应该是别的设备修改过，所以又上传了。可以手动再删除再同步
